src/screens/home/components/subreddits/Subreddit.jsx

```diff
@@ -5,8 +5,12 @@ const Subreddit = (props) => {
 
     const {title, icon_img} = props.subreddit
 
+    const handleOnClick = () => {
+        props.onClick(title.toLowerCase().replace(' ','_'))
+    }
```

Comment:
do this in oneline 
`const handleOnClick = () => props.onClick(title.toLowerCase().replace(' ','_'));`
and implement React.useCallback
Read React Documentation about useCallback and we you should implement here.